q2_feature_table/_core_features/core_features_assets/index.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h1>Core features</h1>
+
+<div class='row'>
+  <div class='col-md-6'>
+    <div class='row'>
+      <div class='col-md-12'>
+        <h3>Feature table summary</h3>
+        Sample count: {{num_samples}}<br>
+        Feature count: {{num_features}}
+      </div>
+    </div>
+    <div class='row'>
+      <div class='col-md-12'>
+        <img src='core-feature-counts.svg'>
+      </div>
+    </div>
+  </div>
+  <div class='col-md-6'>
```

Comment:
Discussed offline with @ebolyen, we're leaving the page layout as-is because the plot is intended to be the first thing users see, and the table provides a more detailed view of what's in the plot. The reflow for small screens also works well as-is because the plot is placed at the top of the page.